src/components/AddBook.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const AddBook = () => (
+  <div>
+    <h2 className="form-title"> ADD NEW BOOK</h2>
+    <form className="add-form">
+      <input className="form-item add-title" type="text" placeholder="Book title" />
```

Comment:
The form in the `AddBook` component should have input for the name of the author as well. The category part is optional. But the name of the author is a part of the requirements for this project. 
![image](https://user-images.githubusercontent.com/33268927/209428228-12ac4ba2-b452-4bf7-9be1-e7ba09ff4bef.png)